vignettes/geologic-map.Rmd

```diff
@@ -100,95 +100,74 @@ library(sf)
+Those *Edmontosaurus* occurrences are now plotted on the map. You may notice that there are some occurrences that don't overlay the plotted outcrop, despite apparently being from the Hell Creek. This could be due to an error in the PBDB data, potentially relating to the accuracy of the latitude and longitude co-ordinates, or it could be that we're missing some outcrop data of the Hell Creek. Either way, this shows that plotting occurrences over their respective outcrop can a good way to explore potential data issues. 
 
 Asides from those errant occurrences, it's starting to look pretty good! But we can add a few more finishing touches to really make it pop. First, let's the use package [`rphylopic`](https://rphylopic.palaeoverse.org/index.html) to add a silhouette of *Edmontosaurus* to the map. 
 
-```{r add rphylopic, eval = TRUE}
+```{r add-rphylopic, eval = TRUE}
 # Load the rphylopic package
 library(rphylopic)
 # Get an appropriate silhouette for Edmontosaurus
-ed.uuid <- get_uuid(name = "Edmontosaurus", n = 3)[[3]]
-# Get the credit for the author of the image
-get_attribution(uuid = ed.uuid)
-# Setup data.frame for the silhouette
+ed_uuid <- get_uuid(name = "Edmontosaurus", n = 3)[3]
+# Get the attribution information for the author of the image
+get_attribution(uuid = ed_uuid, text = TRUE)
+# Setup data.frame for the silhouette (plot location)
 silhouette_df <- data.frame(x = -99, y = 49.5, name = "Edmontosaurus")
 # Plot the map
 ggplot() +
   geom_sf(data = n_a) + 
   geom_sf(data = ca) +
-  geom_sf(data = hc,
-          color = '#a75224',
-          fill = '#C7622B',
-          lwd = 0) +
-  geom_sf(data = ed_sf, 
-          color = 'black', 
-          size = 0.7) +
+  geom_sf(data = hc, color = '#a75224', fill = '#C7622B', lwd = 0) +
+  geom_sf(data = ed_sf, color = 'black', size = 0.7) +
   coord_sf(xlim = c(-112, -97), ylim = c(44, 50), expand = FALSE) +
-  geom_phylopic(data = silhouette_df, aes(x = x, y = y), 
-              uuid = ed.uuid, 
-              size = 0.8, 
-              alpha = 1, 
-              color = "black") +
-  labs(x = "", y = "")
-
+  geom_phylopic(data = silhouette_df, aes(x = x, y = y), uuid = ed_uuid, 
+                size = 0.8, alpha = 1, color = "black") +
+  labs(x = "Longitude (º)", y = "Latitude (º)")
 ```
 
-Nice! Finally, we can add a few other touches to make it a publication worthy figure, including an inlay map and a scale bar:
+Nice! Finally, we can add a few other touches to make it a publication-worthy figure, including an inlay map, a scale bar, and a North arrow:
 
-``` {r add inlay and scale, evail = TRUE}
+``` {r add-inlay-scale, evail = TRUE}
 # Load `ggpatial` package to add annotations, and `cowplot` to stitch final map 
 # together
 library(ggspatial)
```

Comment:
I think this is the only place you use piping. Unless necessary, I would just move the `n_a` object inside the `ggplot()` call and remove any piping.

Reply:
Done.